chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,922 @@
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/9804/measures",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"9804",
+										"measures"
+									]
+								},
+								"description": "GET /{certifiedProductId}/measures end point returns response status 200 for valid API Key and valid certifiedProductId"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{certifiedProductId}/measures - Invalid API Key",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{certifiedProductId}/measures end point should return status 401 for Invalid API Key\", function () {\r",
+											"    pm.response.to.have.status(401);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"value": "1290y7y8h8ko90b8ecd0596c98ae9094",
+										"type": "text"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/10295/measures",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"10295",
+										"measures"
+									]
+								},
+								"description": "GET /{certifiedProductId}/measures end point should return 401 response for Invalid API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{certifiedProductId}/measures - without API Key",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{certifiedProductId}/measures end point should return status 400 for requests without API Key\", function () {\r",
+											"    pm.response.to.have.status(400);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/9544/measures",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"9544",
+										"measures"
+									]
+								},
+								"description": "GET /{certifiedProductId}/measures should return 400 status for requests without API Key"
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "GET : /{chplPrefix}-{identifier}",
```

Comment:
I thought this prefix-identifier thing was part of the data-driven tests; why is it here in the non-data-driven tests?

Reply:
Fixed.